Bottom: 05cb9deb03b6db173916c7cd2da65ade580f0f2f
Top:    59354f1d3aa16f538d85d39f7fcedecd299357a3
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-06-05 12:45:46 +0100

Revert some "True" -> "true" string changes

These were inadvertently changed by the mechanised Boolean -> bool
replacement in commit ad5eb5a165442be38624aa622beeb57dc1d2afbd


---

diff --git a/src/crosshair.c b/src/crosshair.c
index 88b9cc0..13ddc71 100644
--- a/src/crosshair.c
+++ b/src/crosshair.c
@@ -940,7 +940,7 @@ FitCrosshairIntoGrid (LocationType X, LocationType Y)
 	if (ans == NO_TYPE)
 	  hid_action("PointCursor");
 	else if (!TEST_FLAG(SELECTEDFLAG, (LineType *)ptr2))
-	  hid_actionl("PointCursor","true", NULL);
+	  hid_actionl("PointCursor","True", NULL);
     }
   if (Settings.Mode == LINE_MODE
       && Crosshair.AttachedLine.State != STATE_FIRST
diff --git a/src/hid/lesstif/main.c b/src/hid/lesstif/main.c
index bc8f17c..3280032 100644
--- a/src/hid/lesstif/main.c
+++ b/src/hid/lesstif/main.c
@@ -2020,7 +2020,7 @@ lesstif_parse_arguments (int *argc, char ***argv)
 	    break;
 	  case HID_Boolean:
 	    o->argKind = XrmoptionNoArg;
-	    o->value = "true";
+	    o->value = "True";
 	    acount++;
 	    break;
 	  default:
